### ISSUE TITLE
Need $Server->sub URLDecode.

### ISSUE THREAD

**Issue by mamurphy** [omniti-labs/mungo#5]
**Reported by jesus on 1 Jan 1970 00:19 UTC**
```
sub URLDecode {
  my $self = shift;
  my $s = shift;
  $s =~ s/%([a-fA-F0-9]{2})/chr(hex($1))/eg;
  return $s;
}
```

**Comment by mamurphy** [comment]
**Comment by jesus on 1 Jan 1970 00:19 UTC**
(In [12]) Need a decode function, closes #5

**Comment by mamurphy** [comment]
**Modified by clinton on 1 Jan 1970 00:20 UTC**
